perfact/zodbsync/zodbsync.py

```diff
@@ -461,7 +463,9 @@ def fs_read(self, path, encoding=None):
             # Translate file system data
             meta = dict(fix_encoding(meta, encoding))
 
-        return meta
+        # to compare zodb and fs object meta we need to decode them the same
+        # way
+        return meta, meta_str.decode('utf-8')
```

Comment:
There is an `else` part above where `meta_str` is not set, namely if no `__meta__` is present and we fall back to assuming this is a `Folder`. Not sure if this situation is tested yet, but this will break then.